### PR TITLE
docs: add guide for uploading packs to oci registries

### PR DESCRIPTION
**Pull request by caroldelwing**
## Describe the Change

This PR adds the following pages:
- An intro page called _Add a Pack to an OCI Registry_.
- A how-to guide that explains how to upload packs to a Basic OCI registry.
- A how-to guide that explains how to upload packs to an ECR registry.

Additionally, this PR adds _ORAS_ to Vale and the `tar` command to the exceptions list of the commands' long-form Vale rule.

## Changed Pages

💻 [Add a Pack to an OCI Registry](https://deploy-preview-2541--docs-spectrocloud.netlify.app/registries-and-packs/registries/oci-registry/add-pack-oci/)
💻 [Add a Pack to a Basic OCI Registry](https://deploy-preview-2541--docs-spectrocloud.netlify.app/registries-and-packs/registries/oci-registry/add-pack-oci/add-pack-oci-basic)
💻 [Add a Pack to an ECR Registry](https://deploy-preview-2541--docs-spectrocloud.netlify.app/registries-and-packs/registries/oci-registry/add-pack-oci/add-pack-oci-ecr)

## Jira Tickets

🎫 [DOC-1038](https://spectrocloud.atlassian.net/browse/DOC-1038)

## Backports

Can this PR be backported?

- [X] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1038]: https://spectrocloud.atlassian.net/browse/DOC-1038?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+<TabItem label="ECR" value="ECR_Registry">
+
+6. In your terminal, navigate to the directory containing the folder with the pack files.
+
+7. Compress the contents of the pack folder into an archive file. Issue the command below to create the archive file.
+   Replace `<Your_Packs_Folder_Name>` with the name of the folder containing the pack files.
+
+   ```bash
+   tar -czvf $NAME-$VERSION.tar.gz <Your_Packs_Folder_Name>
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md:164]: No long form is available :) I will include `tar` in the list of exceptions of the longform rule.

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+- Palette's tenant admin access to add a new registry.
+
+- Custom pack files available on your local machine. Refer to the [Add an Add-on Pack](../../adding-add-on-packs.md)
+  guide to learn how to create a custom pack.
+
+- A private OCI registry such as [Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/ecr/) or
+  [Harbor](https://goharbor.io/). Learn how to set up a Harbor registry server using the
+  [Harbor Installation and Configuration](https://goharbor.io/docs/2.9.0/install-config/) guide.
+
+- [ORAS](https://oras.land/docs/installation/) and [Tar](https://www.gnu.org/software/tar/) installed on your machine.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md:32]: Excellent point! I completely forgot about this issue.

[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md:32]: :white_check_mark:

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+hide_table_of_contents: false
+sidebar_position: 60
+---
+
+Palette supports the use of Open Container Initiative (OCI) registries. You can register a private OCI registry with
+Palette, publish custom packs, and then use the packs in your cluster profiles.
+
+Two types of OCI authentication are available: registries that support basic authentication, such as Harbor, and AWS
+ECR, which is supported as a third-party registry provider. To upload packs to OCI registries, you can use
+[ORAS](https://oras.land/docs/), a CLI tool for pushing and pulling OCI artifacts to and from OCI registries. Refer to
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md:15]: :white_check_mark:

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md:32]: :white_check_mark:

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+   export AWS_ACCESS_KEY_ID=<Your_Access_Key_ID>
+   export AWS_SECRET_ACCESS_KEY=<Your_Secret_Access_Key>
+   export AWS_DEFAULT_REGION=<Your_Default_Region>
+   ```
+
+2. Additionally, export the required variables for creating the ECR repository and pushing the pack.
+
+   - `REPOSITORY_NAME` - Specify a name for the repository that will store the pack files. For example,
+     `spectro-oci-registry`.
+   - `NAME` - The pack's name, which must match the name in the **pack.json** file.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md:73]: :white_check_mark:

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+
+</TabItem>
+
+<TabItem label="Basic" value="Basic_Registry">
+
+6.  In your terminal, export the variables required for creating the Harbor repository and pushing the pack.
+
+    - `HARBOR_PROJECT` - Specify a name for the Harbor project that will store the repositories and the pack files. For
+      example, `spectro-oci-registry`.
+    - `NAME` - The pack's name, which must match the name in the **pack.json** file.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci.md:181]: :white_check_mark:

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+6. Next, export the variables required for creating the Harbor repository and pushing the pack.
+
+   - `HARBOR_PROJECT` - Specify a name for the Harbor project that will store the repositories and the pack files. For
+     example, `spectro-oci-registry`.
+   - `NAME` - The pack's name, which must match the `name` parameter in the **pack.json** file.
+   - `VERSION` - The pack's version, which must match the `version` parameter in the **pack.json** file.
+
+   ```bash
+   export HARBOR_PROJECT=<Your_Repository_Name>
```
Comment (yuliiiah) [inline docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md:76]: Should this be <your-project-name>? I found the _your repository name_ placeholder a bit confusing 😓

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+11. After pushing the pack to the Harbor registry, follow the steps in [Add OCI Packs Registry](./add-oci-packs.md) to
+    add your Harbor registry to Palette.
+
+:::info
+
+Palette automatically synchronizes the registries. However, you can manually trigger the synchronization if needed. From
+the **OCI Registries** page, click the **three-dot Menu** next to the registry name you added and select **Sync**.
+
+:::
```
Comment (yuliiiah) [inline docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md:119]: I think you could indent it to the step 🤔

**Review by yuliiiah — APPROVED** [review]
The instructions worked for me 🎉 I'm leaving some copy edits and leaving out the things we discussed because you know, I know, we both know 😎 Great job Caro!

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+  Authority (CA), you will need the certificate to add the registry to Palette.
+
+## Upload Pack to a Basic OCI Registry
+
+Palette supports all OCI-compliant registries that use basic authentication. This guide uses a
+[Harbor](https://goharbor.io/) registry as an example. Follow the steps described below to set up your Harbor registry,
+push the pack, and configure the registry in Palette.
+
+1. Access the Harbor registry server domain on your web browser and log in using your credentials. If you have kept the
+   default credentials, the username and password are **admin** and **Harbor12345**, respectively.
```
[the remark this answers is not in the document]
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md:47]: Great idea!

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md (hunk 6 above)
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md]: Ooops. My bad! Fixed :)

File: docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md (hunk 7 above)
Comment (caroldelwing) [reply docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md:119]: :white_check_mark:

**Review by karl-cardenas-coding — APPROVED** [review]
Great additions, thank you @caroldelwing 
Much more simpler to follow

**Pull request merged**